src/compiler/prioritization-process.md

```diff
@@ -0,0 +1,197 @@
+# Prioritization Process
+
+The prioritization WG has a process that iterates every two weeks at the end of which we fill up a document with [meeting notes](https://hackmd.io/pHb6eTZ2Sjy6XZmwXZHIBA?view) of `T-compiler` working group (TODO: what is the correct cadence of these meetings?).
```

Comment:
The priorit... meetings are biweekly, compiler team ones are weekly, correct? how do they interwine?